README.md

```diff
@@ -45,51 +45,48 @@ data science with BigQuery.
 The easiest way to run the VCF to BigQuery pipeline is to use the
 [docker](https://www.docker.com/) image and run it with the
 [Google Genomics Pipelines API](https://cloud-dot-devsite.googleplex.com/genomics/pipelines)
-as it has the binaries and all dependencies pre-installed.
+as it has the binaries and all dependencies pre-installed. Please ensure you
+have the latest `gcloud` tool by running `gcloud components update` (more
+details [here](https://cloud.google.com/sdk/gcloud/reference/components/update)).
 
-First, set up the pipeline configurations shown below and save it as
-`vcf_to_bigquery.yaml`. The parameters that you need to replace are:
+Run the script below and replace the following parameters:
 
-* `my_project`: This is your project name that contains the BigQuery dataset.
-* `gs://my_bucket/vcffiles/*.vcf`: A location in Google Cloud Storage where the
+* `PROJECT_ID`: This is your project ID that contains the BigQuery dataset.
```

Comment:
Optional, but cloud shell automatically defines GOOGLE_CLOUD_PROJECT so I like to use it.  It's longer, obviously, though.  This is also more relevant in the cloud.google.com/ documentation since in that context it's very easy to spawn thw cloud shell.

Reply:
sg! updated to GOOGLE_CLOUD_PROJECT.